### ISSUE TITLE
Confusion about the formula of content adversarial loss

### ISSUE THREAD

**Issue by Victarry** [HsinYingLee/DRIT#56]
In the original paper, the content adversarial loss is:
![image](https://user-images.githubusercontent.com/23012959/120264126-b4aa2380-c2cf-11eb-8e7a-651c047b51fb.png)

However, according to the code:
```python
def backward_contentD(self, imageA, imageB):
    pred_fake = self.disContent.forward(imageA.detach())
    pred_real = self.disContent.forward(imageB.detach())
    for it, (out_a, out_b) in enumerate(zip(pred_fake, pred_real)):
      out_fake = nn.functional.sigmoid(out_a)
      out_real = nn.functional.sigmoid(out_b)
      all1 = torch.ones((out_real.size(0))).cuda(self.gpu)
      all0 = torch.zeros((out_fake.size(0))).cuda(self.gpu)
      ad_true_loss = nn.functional.binary_cross_entropy(out_real, all1)
      ad_fake_loss = nn.functional.binary_cross_entropy(out_fake, all0)
    loss_D = ad_true_loss + ad_fake_loss
    loss_D.backward()
    return loss_D
```

I think this formula should be written as:
![image](https://user-images.githubusercontent.com/23012959/120264727-f5566c80-c2d0-11eb-9b7e-08e9288f5f31.png)
After all, the discriminator get optimal when it outputs 0.5 for all input in the situation of original formula

**Comment by Wolfybox** [comment]
Duplicate Issue: #17
